components/auction_reminders/view.coffee

```diff
@@ -22,29 +22,43 @@ module.exports = class AuctionReminderView extends Backbone.View
       e.preventDefault()
       return @close()
 
-  postRender: ->
-    @clock = new ClockView
-      el: @$('.js-auction-reminder-clock')
-      model: @model
-      modelName: 'Auction'
-      stateCallback: =>
-        @close()
-
-    @clock.start =>
-      @$el.attr 'data-state', 'open'
-
-  render: ->
-    @$el.html template
-      auction: @model
-      _s: _s
-    @postRender()
+  getOffsetTimesPromise: ->
+    Q.promise (resolve, reject, notify) =>
+      @model.calculateOffsetTimes
+        success: =>
+          switch @model.reminderStatus()
+            when 'live_open_soon'
+              startMoment = @model.get('offsetLiveStartAtMoment')
+              # only show minutes if auction doesn't start on the hour
+              # & show add 'tomorrow' if auction starts tomorrow
+              formatStr = if startMoment.minutes() == 0 then 'hA' else 'h:mmA'
+              formattedTime = startMoment.format(formatStr)
+              time = if startMoment.isSame(moment(), 'day') then formattedTime else "#{formattedTime} Tomorrow"
+              @$el.find('.time-msg').html("Auction begins at #{time}")
+            when 'closing_soon'
+              endMoment = @model.get('offsetEndAtMoment')
+              time = endMoment.fromNow()
+              @$el.find('.time-msg').html("Auction ends #{time}")
+          resolve(this)
+
+
+  preRender: ->
```

Comment:
I'm really not crazy about how I named this... the reason is that there isn't a proper auction reminders collection that is handling the rendering exactly, so it renders the html string of each component and then appends it to the dom in index.coffee's #attach method above. on top of that it returns a deferred/Promise so I wasn't sure the best way to approach it. I chose not to use the standard `render` name.